crates/cli_testing_examples/benchmarks/platform/host.zig

```diff
@@ -210,7 +210,8 @@ fn roc_fx_getInt_help() !i64 {
     const stdin = std.io.getStdIn().reader();
     var buf: [40]u8 = undefined;
 
-    const line: []u8 = (try stdin.readUntilDelimiterOrEof(&buf, '\n')) orelse "";
+    const raw_line: []u8 = (try stdin.readUntilDelimiterOrEof(&buf, '\n')) orelse "";
+    const line = std.mem.trimRight(u8, raw_line, &std.ascii.spaces);
```

Comment:
this deals with `\r` in the line

Reply:
Worth adding this as a code comment too, to save someone from trying to "optimize" this later.

Reply:
will do in a follow-up that makes a couple more tests pass